scripts.py

```diff
@@ -699,6 +700,9 @@ def facets(self, lane):
 
 class CacheMARCFiles(LaneSweeperScript):
```

Comment:
This could be a timestamp script, although it's a little weird since most of the time it runs it doesn't do anything.